plugins/admin/pkg/command/autoscaling/set.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package autoscaling
+
+import (
+	"fmt"
+	"os"
+
+	"knative.dev/client-contrib/plugins/admin/pkg"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+)
+
+var (
+	ScaleToZero       = ""
+	enableScaleToZero = "enable-scale-to-zero"
+	knativeServing    = "knative-serving"
+	configAutoscaler  = "config-autoscaler"
+)
+
+func NewAutoscalingSetCommand(p *pkg.AdminParams) *cobra.Command {
+	AutoscalingSetCommand := &cobra.Command{
+		Use:   "set",
+		Short: "set autoscaling config",
+		Long: `set autoscaling config provided by Knative Pod Autoscaler (KPA)
+
+For example:
+# To enable or disable scale-to-zero
+kn admin autoscaling set --scale-to-zero true
+`,
+		Run: func(cmd *cobra.Command, args []string) {
+			currentCm := &corev1.ConfigMap{}
```

Comment:
name this `currentCmd` not `currentCm` which could be confusing later

Reply:
as it is short for current ConfigMap, so I think we can keep it as `currentCm`